<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RERestDishComplex</class>
 <widget class="QDialog" name="RERestDishComplex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Complex dish</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="EQLineEdit" name="leNameEn">
           <property name="Field" stdset="0">
            <string>f_en</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="EQLineEdit" name="leCode">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="Field" stdset="0">
            <string>f_id</string>
           </property>
           <property name="PrimaryKey" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="EQLineEdit" name="leNameAm">
           <property name="Field" stdset="0">
            <string>f_am</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="EQLineEdit" name="lePrice">
           <property name="Field" stdset="0">
            <string>f_price</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="EQTimeEdit" name="teStart">
           <property name="Field" stdset="0">
            <string>f_startTime</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="EQTimeEdit" name="teEnd">
           <property name="Field" stdset="0">
            <string>f_endTime</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="EQLineEdit" name="lePriceDeviation">
           <property name="Field" stdset="0">
            <string>f_priceDeviation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name, am</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Name, en</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Start time</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>End time</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="EQLineEdit" name="leNameRu">
           <property name="Field" stdset="0">
            <string>f_ru</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="EQLineEdit" name="leAdgt">
           <property name="Field" stdset="0">
            <string>f_adgt</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Name, ru</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Price
deviation</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>ADGT</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="EQCheckBox" name="chEnabled">
           <property name="text">
            <string/>
           </property>
           <property name="Field" stdset="0">
            <string>f_enabled</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>List of dishes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="EQLineEdit" name="leMenu">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Menu</string>
            </property>
            <property name="ShowButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leMenuCode"/>
          </item>
          <item>
           <widget class="EQLineEdit" name="leDishCode"/>
          </item>
          <item>
           <widget class="EQLineEdit" name="leDishName">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Dish</string>
            </property>
            <property name="ShowButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resort/res.qrc">
              <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="EQTableWidget" name="tblDishes">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <column>
          <property name="text">
           <string>RecId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MenuId</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Menu</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>354</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EQLineEdit" name="leDishesTotal">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>86</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">eqcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>EQTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header location="global">eqtimeedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
